main.js

```diff
@@ -0,0 +1,70 @@
+const booksElement = document.querySelector('.books');
+const titleInput = document.querySelector('.title-input');
+const authorInput = document.querySelector('.author-input');
+const addBookBtn = document.querySelector('#addBtn');
+
+let books = [];
+
+const saveToLocalStorage = (books) => {
+  localStorage.setItem('books', JSON.stringify(books));
+};
+
+const getExistingBooks = () => JSON.parse(localStorage.getItem('books'));
+
+function generateRandomId() {
+  return Math.random().toString(20).substr(2, 20);
+}
+
+const addBooks = () => {
+  const title = titleInput.value;
+  const author = authorInput.value;
+  const id = generateRandomId();
+
+  const newBook = { title, author, id };
+
+  if (getExistingBooks()) {
+    getExistingBooks().forEach((existingBook) => {
+      books.push(existingBook);
+    });
+  }
+
+  books.push(newBook);
```

Comment:
[OPTIONAL] Consider using methods that don't mutate the array.

<img width="407" alt="Screen Shot 2022-06-27 at 16 48 36" src="https://user-images.githubusercontent.com/82426682/176041544-f6a4635f-dec5-41b4-9f6f-dd3bc6511c6e.png">